board/piksiv3/rootfs-overlay/etc/init.d/S84piksi_fpga

```diff
@@ -5,13 +5,22 @@ name="piksi_fpga"
 start() {
   bit="/lib/firmware/piksi_fpga.bit"
   if [ -f "$bit" ]; then
-    cat "$bit" > /dev/xdevcfg
-    prog_done=`cat /sys/devices/soc0/amba/f8007000.devcfg/prog_done`
-    if [ "$prog_done" != "1" ]; then
-      echo "ERROR: configuration failed"
-    fi
+    tries=1
+    while [ $tries -le 3 ]; do
```

Comment:
Need to increment tries somewhere

Reply:
omg.. am I tired.. I started with a `for`, ended with a `while`, forgot the increment :(